README.md

```diff
@@ -26,10 +26,15 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Cedar relat
   - [How Cedar was built with verification-guided development](https://www.amazon.science/blog/how-we-built-cedar-with-automated-reasoning-and-differential-testing)
   - [Cedar: A new policy language](https://onecloudplease.com/blog/cedar-a-new-policy-language)
   - [Scaling Authorization with Cedar and OPAL](https://www.permit.io/blog/scaling-authorization-with-cedar-and-opal)
+  - [Using Open Source Cedar to Write and Enforce Custom Authorization Policies](https://aws.amazon.com/blogs/opensource/using-open-source-cedar-to-write-and-enforce-custom-authorization-policies/)
+  - [https://www.amazon.science/blog/how-we-built-cedar-with-automated-reasoning-and-differential-testing](https://www.amazon.science/blog/how-we-built-cedar-with-automated-reasoning-and-differential-testing)
+  - [The Cedar Programming Language: Authorization Simplified](https://thenewstack.io/the-cedar-programming-language-authorization-simplified/)
+  - [Verification-Guided Development of the Cedar Authorization Language](https://cps-vo.org/node/94470)
```

Comment:
The HCSS page isn't an article, it's an abstract w/ slides. You could include it in a "talks" section, or just leave it out.